MediaBrowser.Controller/Entities/TV/Season.cs

```diff
@@ -102,10 +102,11 @@ public Series Series
             get
             {
                 var seriesId = SeriesId;
-                if (seriesId.Equals(Guid.Empty)) {
+                if (seriesId == Guid.Empty)
+                {
                     seriesId = FindSeriesId();
                 }
-                return !seriesId.Equals(Guid.Empty) ? (LibraryManager.GetItemById(seriesId) as Series) : null;
+                return seriesId == Guid.Empty ? null : (LibraryManager.GetItemById(seriesId) as Series);
```

Comment:
Can seriesId be null? It might not work then. C# whatever the new one is has some stuff for this, but I'm not sure.

Reply:
No `Guid` should ever be null, only empty.